Serenity updates - October 2020 - Momo's news (#786)
akeneo/pim-helpcenter: content/updates/2020-10/3-Better-manage-product-models-and-variants-in-the-grid.md

---
pim_announcement_alt_img: "TODO"
pim_announcement_audience:
- CE
- EE
---

# Better manage product models and variants in the grid

> You work a lot with product models and their variants? You will enjoy these simple yet efficient new improvements we delivered in the product grid.

Author: We can't mention the "Read more" button as this text is also shown in the help center. I would suggest:
```suggestion
Do you work a lot with product models and variants? If yes, you will surely enjoy these simple but yet efficient improvements we made in the product grid: a new filter on the product type, a new display option...
```

## Filter on the product typology
::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/enrich-products-variants.html#filter-on-product-typology"
:::

The new `Product typology` filter in the product grid allows you to choose whether to display only the simple products or only the variant products. Pretty useful when you want to quickly find different variant products in different product models, for instance.

![](../img/TODO.png)

## Product variant display option
::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/TODO.html"
:::

The product grid also benefits from a new display option: the `Variant` display option. It allows you to show your models' product variants, instead of only these product models. This way, you can easily find and work at your product variant level. What a time saver!

![](../img/TODO.png)

## Filter on the product identifier
::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/TODO.html"
:::

> Filtering on your product model identifiers has now arrived in your product grid! So neat!

Author: Only PM Id.

> Filtering on your product model identifiers has now arrived in your product grid! So neat!

Author: I'm not sure it concerns the search bar. @morganen Does is work in the search bar too?

> Filtering on your product model identifiers has now arrived in your product grid! So neat!

Reviewer: So it applies to the system filter "Identifier" and in the search bar (I could filter on a PM ID through the search bar)?

> Filtering on your product model identifiers has now arrived in your product grid! So neat!

Reviewer 2: Not only PM: you can search on the sub PM identifier as well. And yes, it works also with the search bar :)

> Filtering on your product model identifiers has now arrived in your product grid! So neat!

Author: Ok, so it's both thanks to the search bar and the identifier filter. In this case, I would change the title. And I think we need to change your paragraph, Laure, so that we can mention it.
```suggestion
You can now filter on your product model (and sub-product model) identifiers in your product grid! Enter your product model ID in our search bar or in the `Identifier` filter et voilà!
```
@LaureBro what do you think?

> Filtering on your product model identifiers has now arrived in your product grid! So neat!

Reviewer 2: Update : 
- le filtre permet de chercher sur l'identifier d'un product model ou d'un sous-produit model, ou d'un produit (simple ou variant. 
- dans la search bar, on peut chercher sur l'identifiant (= code) ou sur le label d'un product model, d'un sous-produit model et d'un produit (variant ou non). 
La seule nouveauté réside dans le fait que le nouveau filtre remonte aussi les parents.

---
pim_announcement_img: ./img/better-manage-product-models-and-variants-in-the-grid.png
pim_announcement_alt_img: "New counters and display option in the product grid"
pim_announcement_audience:
- CE
- EE
---

# Better manage product models and variants in the grid

Do you work a lot with product models and variants? If yes, you will surely enjoy these simple but yet efficient improvements we made in the product grid: a new filter on the product type, a new display option...

## Filter on the product typology
::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/enrich-products-variants.html#filter-on-product-typology"
:::

> The new `Product typology` filter in the product grid allows you to choose to display only the simple products or only the variant products. It's pretty useful if you need to quickly find different variant products in different product models, for instance.

Reviewer 2: En fait, en sélectionnant "variant", on peut afficher soit les PM uniquement (si l'option "groupée" est activée), soit les variants uniquement (si l'option "dégroupée" est activée). Je mettrais peut être ça après le "product variant display option" du coup.

Author: Which feature is the most important one? The most expected? The filter on typology or the display option? @morganen

Reviewer 2: They are both very expected and important... Hard to say, they are complementary.

![Product typology filter](../img/product-typology-filter.png)

> ## Product variant display option

Reviewer 2: c'est pas "variant product display option" plutôt?

Author: Hm yes, you're right.
```suggestion
## Variant product display option
```

> The new `Product typology` filter in the product grid allows you to choose to display only the simple products or only the variant products. It's pretty useful if you need to quickly find different variant products in different product models, for instance.

Reviewer: ```suggestion
The new `Product typology` filter in the product grid allows you to choose to display only the simple products or only the variant products (or only their product models if you group your variants).```

::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/enrich-products-variants.html#display-products-with-variants"
:::

The product grid also benefits from a new display option: the `Variant` display option. Use it if you want to list your product variants in the grid, instead of listing their product models. Thanks to this new option, you do not need anymore to first click on a product model to access its variants. What a time saver!

![Product variant display option](../img/product-variant-display-option.png)

> ## Filter on the product identifier

Author: ```suggestion
## Search product models with their identifier
```

Author: @LaureBro What do you think? Sounds way clearer to me.

Reviewer: It's clearer, and what about "using" instead of with?
```suggestion
## ## Search product models using their identifier
```

Author: Yup, I'm in. Removing the double ##:
```suggestion
## Search product models using their identifier
```

> ## Filter on the product identifier

Reviewer: It's clearer, and what about "using" instead of with?
```suggestion
## ## Search product models using their identifier
```

> The product grid also benefits from a new display option: the `Variant` display option. Use it if you want to list your variant products in the grid, instead of listing their product models. Thanks to this new option, you do not need anymore to first click on a product model to access its variants. What a time saver!

Reviewer: ```suggestion
The product grid also benefits from a new display option `Variant` at the top of the grid. Use it if you want to list your variant products in the grid, or keep them grouped by product models. Thanks to this new option, you do not need anymore to first click on a product model to access its variants. What a time saver!```

Author: I kept your suggestion but fixed some typos:
```suggestion
The product grid also benefits from a new `Variant` display option at the top of the grid. Use it if you want to list your variant products in the grid, or keep them grouped by product models. Thanks to this new option, you do not need anymore to first click on a product model to access its variants. What a time saver!
```
